db/migrate/20200901081623_create_authors.rb

```diff
@@ -0,0 +1,10 @@
+class CreateAuthors < ActiveRecord::Migration[6.0]
+  def change
+    create_table :authors do |t|
+      t.string :username
```

Comment:
- [ ] rename this column to `name` not `username`

Reply:
The authors.rb has been deleted already! There is no file like that so from where you're looking for we did not understand.